src/jquery.eclipsefdn-api.js

```diff
@@ -1795,6 +1795,7 @@
         "host": window.location.host,
         "source": window.location.pathname,
         "publish_to": $container.data("publish-target"),
+        "campaign_name": $container.data("campaign-name")
```

Comment:
I am not convinced this is the right solution. We won't know the campaign_name once this is on production.

Let's use the node_id:

Fetch ads;
https://newsroom.eclipse.org/api/ads/:nid

We will need to be able to display specific ads from time to time. For example, we could use this to show a preview to the Membership team when creating a new node.

I think we will need to be able to make 2 different calls to our API:

- api/ads (current one)
- api/ads/:nid - This would be used when we want to display a specific ad.

the code in singlePromo could be something like:

```
// @todo: validate that ad-id is a numerical value + not empty
if ($container.data("ad-id")) {
 $.ajax(...api/ads/:nid, {
        dataType: "json",
        contentType: "application/json",
        type: "PUT"....
}
else {
 $.ajax(...api/ads/, {
        dataType: "json",
        contentType: "application/json",
        type: "post"....
}

```